objstore.go

```diff
@@ -442,6 +442,13 @@ func WrapWithMetrics(b Bucket, reg prometheus.Registerer, name string) *metricBu
 			Name: "objstore_bucket_last_successful_upload_time",
 			Help: "Second timestamp of the last successful upload to the bucket.",
 		}, []string{"bucket"}),
+
+		opsWrittenBytes: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+			Name:        "objstore_bucket_operation_written_bytes",
+			Help:        "Number of bytes uploaded from TSDB block, per operation.",
+			ConstLabels: prometheus.Labels{"bucket": name},
+			Buckets:     prometheus.ExponentialBuckets(50<<100, 2, 10),
+		}, []string{"operation"}),
```

Comment:
I think it is better rather than using a separate metric, use the transferred bytes histogram vec metric, as it is named so that it allows to measure up and downloads and then the label `operation` would determine if it is in fact an upload or download.

Reply:
Removed the metric and will work on the remaining comments.